apps/api-documenter/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./node_modules/@microsoft/rush-stack-compiler-3.5/includes/tsconfig-node.json",
+  "extends": "./node_modules/@rushstack/heft-node-rig/profiles/library/tsconfig-base.json",
```

Comment:
Why is **api-documenter** using `/profiles/library/`? 

It is a CLI tool.

But it does also export a plugin API. Hmm

Reply:
I see -- it is a cyclic dependency